src/parse-stream.js

```diff
@@ -325,6 +325,18 @@ export default class ParseStream extends Stream {
       });
       return;
     }
+    match = (/^#EXT-X-PROGRAM-DATE-TIME:?((\d{4})-((0[1-9])|(1[0-2]))-((0[1-9])|(([1-2]\d)|(3[0-1])))T(([0-1]\d)|(2[0-3])):([0-5]\d):([0-5]\d)(\.(\d{1,3}))?((\+|-)((0\d)|(1[0-2])):([0-5]\d))?)?$/).exec(line);
+    if (match) {
+      event = {
+        type: 'tag',
+        tagType: 'date-time'
+      };
+      if (match[1]) {
+        event.dateTime = match[1];
```

Comment:
`Date.parse` will only return the timestamp right?
Then I guess I can understand what @pswolfe means, but I can also understand asking the users to `Date.parse` it every time is also cumbersome...
We could always add both?